package.json

```diff
@@ -7,10 +7,11 @@
     "format": "biome format ./",
     "lint:fix": "biome lint --fix ./",
     "lint": "biome lint ./",
+    "fix-commit": "node --run format:fix; node --run lint:fix",
     "pre-commit": "node --run lint:fix; node --run format:fix; node --run test:types; node --run test:unit; node --run test:e2e",
-    "test:e2e": "node --no-warnings --experimental-test-coverage --test-reporter=lcov --test-reporter-destination=./coverage.lcov --test-reporter=spec --test-reporter-destination=stdout --import './build/snapshots.ts' --test --test-coverage-include='recipes/**/*' --test-coverage-exclude='**/*.e2e.{mjs,mts}' './packages/*/*.e2e.{mjs,mts}'",
+    "test:e2e": "NODE_OPTIONS='--no-warnings --experimental-test-coverage --test-reporter=spec --test-reporter-destination=stdout --import=\"./build/snapshots.mts\" --test-coverage-include=\"recipes/**/*\" --test-coverage-exclude=\"**/*.e2e.{mjs,mts}\"' npm run test:e2e --workspaces",
```

Comment:
would it make more sense to put these in an env file, and pass only the env file option in all these places instead?

Reply:
Yes! But only if we can figure out the problem where options we need are disallowed within `NODE_OPTIONS`.

Reply:
https://github.com/nodejs/node/pull/56114

Reply:
I think this is obsolete with the new way of orchestrating this